src/applications/personalization/profile360/actions/index.js

```diff
@@ -64,7 +64,14 @@ export function fetchAddressConstants() {
 export function fetchTransactions() {
   return async (dispatch) => {
     try {
-      const response = isVet360Configured() ? await apiRequest('/profile/status/') : localVet360.getUserTransactions();
+      let response;
+      if (isVet360Configured()) {
+        response = await apiRequest('/profile/status/');
+      } else {
+        response = { data: [] };
+        // Uncomment the line below to simulate transactions being processed during initialization
+        // response = localVet360.getUserTransactions();
+      }
```

Comment:
This is configured to succeed every time now so that E2E tests run consistently.